stubs/app/Http/Livewire/ResourceSelect.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Http\Livewire;
+
+use App\Models\Resource;
+use App\Models\ResourceCollection;
+use Livewire\Component;
+use Illuminate\Database\Eloquent\Collection;
+
+class ResourceSelect extends Component
+{
+    public Collection $availableResources;
+    public Collection $selectedResources;
+    public string $message = '';
+
+
+    public function mount(?int $resourceCollectionId)
+    {
+        $this->availableResources = new Collection();
+        $this->selectedResources = new Collection();
+        if ($resourceCollectionId != null) {
+            $resourcesInCollection = ResourceCollection::find($resourceCollectionId)->resources()->get();
+            $this->availableResources = Resource::all()->diff($resourcesInCollection);
```

Comment:
In the [integration test](https://github.com/fluid-project/hearth/runs/8040218647?check_suite_focus=true#step:11:74) I see this error:

```
Error: Called 'diff' on Laravel collection, but could have been retrieved as a query.
```

A quick search found this: https://laracasts.com/discuss/channels/laravel/called-diff-on-laravel-collection-but-could-have-been-retrieved-as-a-query

As I understand it, the issue here is that it's much less efficient to query ALL resources, then remove some, when one could query resources with the specified IDs instead. So I think you should make the following change:

```suggestion
            $this->availableResources = Resource::whereNotIn($resourcesInCollection->pluck('id'))->get();
```